app.go

```diff
@@ -480,3 +480,11 @@ func (a *Author) String() string {
 
 	return fmt.Sprintf("%v%v", a.Name, e)
 }
+
+// DefaultAppComplete returns an ActionFunc to run a default command if non were passed.
+// Usage: `app.Action = DefaultCommand("command")`
```

Comment:
Because the target of this comment is likely to be end-users, could we use `cli.DefaultCommand("command")`?